spec/helpers/declarable_helper_spec.rb

```diff
@@ -10,6 +10,78 @@
 
   let(:url_options) { { country: 'AR', year: '2021', month: '01', day: '01' } }
 
+  describe '#declarable_stw_link' do
+    subject(:declarable_stw_link) { helper.declarable_stw_link(declarable, search) }
+
+    let(:declarable) { build(:commodity) }
+    let(:geographical_area) { build(:geographical_area, id: 'FR', description: 'France') }
+    let(:search) { Search.new(country: 'FR', day: '01', month: '01', year: '2021') }
+
+    before do
+      allow(GeographicalArea).to receive(:find).and_return(geographical_area)
+    end
+
+    context 'when no date is passed' do
+      let(:search) { Search.new(country: 'FR') }
+
+      it { is_expected.to include("importDateDay=#{Time.zone.today.day}") }
+      it { is_expected.to include("importDateMonth=#{Time.zone.today.month}") }
+      it { is_expected.to include("importDateYear=#{Time.zone.today.year}") }
+    end
+
+    context 'when the declarable is a heading' do
+      let(:declarable) { build(:heading) }
+
+      it 'returns the correct expected text' do
+        expected_text = "Check how to import heading #{declarable.code} from France."
+
+        expect(declarable_stw_link).to include(expected_text)
+      end
+
+      it { is_expected.to include('https://test.com/stw-testing?') }
+      it { is_expected.to include("commodity=#{declarable.code}") }
+      it { is_expected.to include('originCountry=FR') }
+      it { is_expected.to include('goodsIntent=bringGoodsToSell') }
+      it { is_expected.to include('userTypeTrader=true') }
+      it { is_expected.to include('tradeType=import') }
+      it { is_expected.to include('destinationCountry=GB') }
+      it { is_expected.to include('importDeclarations=yes') }
+      it { is_expected.to include('importOrigin=&') }
+      it { is_expected.to include('importDateDay=01') }
+      it { is_expected.to include('importDateMonth=01') }
+      it { is_expected.to include('importDateYear=2021') }
+      it { is_expected.to include('target="_blank"') }
+      it { is_expected.to include('rel="noopener"') }
+      it { is_expected.to include('class="govuk-link"') }
```

Comment:
so many test, maybe we can group some of this? 

for example the 
   ```   it { is_expected.to include('importDateDay=01') }
      it { is_expected.to include('importDateMonth=01') }
      it { is_expected.to include('importDateYear=2021') }
```
 ?

the benefit are less duplication, and faster tests